### PR TITLE
Add org.pydpainter.PyDPainter

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/mriale/PyDPainter/pull/100
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://github.com/mriale/PyDPainter/pull/100

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flatpak.org/en/latest/first-build.html

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build org.pydpainter.PyDPainter

File: org.pydpainter.PyDPainter.desktop
Comment (bbhtt) [inline org.pydpainter.PyDPainter.desktop:1]: This should be upstream. You didn't submit it.

File: org.pydpainter.PyDPainter.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  <description>
+    <p>PyDPainter, pronounced "Pied Painter" (like Pied Piper), is an attempt to create a usable pixel art program in Python using PyGame. The original inspiration came from the Commodore Amiga version of Deluxe Paint released by Electronic Arts in 1985. Back then, Deluxe Paint helped define the user interface of a paint program with tool bars, menus, and the novel use of left and right mouse buttons for painting and erasing. After pixel art gave way to photo-realism and high-resolution 24 bit color, Deluxe Paint was largely forgotten for artistic work -- left behind in the ever-progressing march of technology.</p>
+    <p>Recently, with a resurgence of all things "retro," low-resolution pixel art and limited color palettes have become popular once again. Many tools to deal with this medium are either too complicated or too crude. This project is an attempt to bring back an old but reliable tool and enhance it with some features to help it better coexist in the modern world.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://github.com/mriale/PyDPainter/raw/master/pydpainter-screenshot.png</image>
```
Comment (bbhtt) [inline org.pydpainter.PyDPainter.metainfo.xml:16]: Change it to a tag instead of master

**Comment by bbhtt** [comment]
> Build [78697 successful](https://buildbot.flathub.org/#/builders/6/builds/78697) To test this build, install it from the testing repository:
> 
> ```
> flatpak install --user https://dl.flathub.org/build-repo/61297/org.pydpainter.PyDPainter.flatpakref
> ```

Are the menus in the top bar supposed to disappear after clicking? This makes it pretty impossible to select any option in them.

**Comment by Mailaender** [comment]
Keep the mouse button pressed. I assume this is imitated older UI conventions.

**Comment by Mailaender** [comment]
bot, build org.pydpainter.PyDPainter

**Pull request closed without merging**